src/biogeophys/OzoneMod.F90

```diff
@@ -453,16 +590,25 @@ subroutine CalcOzoneUptakeOnePoint( &
     !
     ! !LOCAL VARIABLES:
     integer  :: dtime          ! land model time step (sec)
-    real(r8) :: dtimeh         ! time step in hours
+    real(r8) :: avg_dayspyr    ! average number of days per year
     real(r8) :: o3concnmolm3   ! o3 concentration (nmol/m^3)
     real(r8) :: o3flux         ! instantaneous o3 flux (nmol m^-2 s^-1)
     real(r8) :: o3fluxcrit     ! instantaneous o3 flux beyond threshold (nmol m^-2 s^-1)
     real(r8) :: o3fluxperdt    ! o3 flux per timestep (mmol m^-2)
     real(r8) :: heal           ! o3uptake healing rate based on % of new leaves growing (mmol m^-2)
-    real(r8) :: leafturn       ! leaf turnover time / mortality rate (per hour)
+    real(r8) :: leafturn       ! leaf turnover time / mortality rate (per sec)
     real(r8) :: decay          ! o3uptake decay rate based on leaf lifetime (mmol m^-2)
 
-    character(len=*), parameter :: subname = 'CalcOzoneUptakeOnePoint'
+    ! o3:h2o resistance ratio defined by Sitch et al. 2007
+    real(r8), parameter :: ko3 = 1.67_r8
```

Comment:
@lifang0209 should any of these be added to the parameter file, so they can be modified rather than hardcoded here?

Reply:
Hi Erik, thanks for your time and suggestions.
The correct (observational) value for ko3 should be  
 real(r8), parameter :: ko3 = 1.51_r8
which used in CalcOzoneUptakeLi2024OnePoint in the same file (OzoneMod.F90).
I'm not sure if it's good to set different values of a parameter for different schemes in the parameter file.